doc/source/guide/custom.rst

```diff
@@ -0,0 +1,123 @@
+********************************************
+Registering Custom Recommendation Actions
+********************************************
+
+.. note:: You can follow along this tutorial in a Jupyter notebook. [`Github <https://github.com/lux-org/lux-binder/blob/master/tutorial/8-custom-action.ipynb>`_] [`Binder <https://mybinder.org/v2/gh/lux-org/lux-binder/master?urlpath=tree/tutorial/8-custom-action.ipynb>`_]
+
+In this tutorial, we will look at how you can register custom recommendation actions (i.e. tabs of recommendations) to display on the Lux widget. The custom actions can be globally defined and used across different dataframes. We look at the `Happy Planet Index <http://happyplanetindex.org/>`_ dataset, which contains metrics related to well-being for 140 countries around the world. 
+
+.. code-block:: python
+
+    df = pd.read_csv("https://github.com/lux-org/lux-datasets/blob/master/data/hpi.csv")
+    lux.config.default_display = "lux" # Set Lux as default display
+    df
+
+As we can see, Lux displays several recommendation actions, such as Correlation and Distributions, which is globally registered by default.
+
+Registering Custom Actions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's define a custom function to generate the recommendations on the dataframe. In this example, we register a custom action called `G10` to generate a collection of visualizations that showcases numerical measures that differs significantly across `G10 <https://en.wikipedia.org/wiki/Group_of_Ten_(economics)>`_ and non-G10 countries. In other words, we want to understand how the G10 and non-G10 countries differs based on the measures present in the dataframe. Here, we first generate a list of visualizations (VisList) that looks at how various quantitative attributes breakdown between G10 and non-G10 countries. Then, we score and rank these visualization by calculating the percentage difference in means across G10 v.s. non-G10 countries.
+
+.. code-block:: python
+
```

Comment:
Here, you give an example of the type of function that could be a registered action, but can you specify what the return type must be? Is there more formating for this feature, or will all functions that follow this return type work? 

Also, may not be in scope for this PR, but what error handling is there?